vignettes/data_dictionary.Rmd

```diff
@@ -176,7 +176,7 @@ plot_table(table)
 
 ## Aggregated PACTA metrics
 
-The aggregated PACTA metrics are also run across all input banking books. The calculations produce the net aggregate alignment metric, which is defined in `pacta.multi.loanbook.analysis` and allows producing the corresponding plots using `pacta.multi.loanbook.plot`. Results are grouped at the level defined by the `by_group` parameter.
+The aggregated PACTA metrics are also run across all input banking books. The calculations produce the net aggregate alignment metric, which is defined in the vignettes ["Calculation of a company alignment metric"](company_alignment_metric.html) and ["Calculation of exposure-weighted aggregated alignment metric"](loanbook_aggregated_alignment_metric.html) and allows producing the corresponding plots. Results are grouped at the level defined by the `by_group` parameter.
```

Comment:
will this `loanbook_aggregated_alignment_metric.html` link to anything? within the package, we do not have the html files, so I am assuming that gitpages understands that this is the fully rendered vignette, even though we do not provide the full url?

Reply:
it works in the pkgdown site (I tested locally) because all the vignettes are rendered to HTML in the same directory, so a "relative path/URL" is sufficient

I could not figure out any other way to link to a vignette other than using e.g. `vignette("data_ditionary")` as I did elsewhere in this PR, which renders in the pkgdown site exactly like that, and I thought the name of the vignette properly spelled out was important here.

Would very much agree to a better/alternate way of doing this, but I couldn't find one in my searching today. 🤷🏻